overlays/NetworkManager/internal/networkmanager_test.go

```diff
@@ -337,6 +342,9 @@ mode=balance-rr
 xmit_hash_policy=layer2+3
 updelay=0
 
+
+[ethernet]
```

Comment:
This is the culprit I am hesitating. We have here a stray [ethernet] line for a bond. @anderbubble whats your take?

Reply:
> It gets messy quickly.

Remember, readability and maintainability is also important.

Reply:
IB Bonding could also be a separate overlay if things git too messy.